functions/src/index.ts

```diff
@@ -1,27 +1,46 @@
-import {config, firestore as database, https} from 'firebase-functions';
+import {config, firestore as firestoreFromFunctions, https} from 'firebase-functions';
 import {Request, Response} from "express";
 import {createProbot, Options} from "probot";
 import {consoleStream, registerTasks, Tasks} from "./util";
-import {credential, firestore, initializeApp} from "firebase-admin";
+import {credential, firestore, initializeApp, app, database} from "firebase-admin";
 import {DocumentSnapshot} from "firebase-functions/lib/providers/firestore";
 import {EventContext} from "firebase-functions/lib/cloud-functions";
+import { HttpClient } from './http';
 
 let tasks: Tasks;
 let probotConfig: Options = config().probot;
+
+let sizeApp: app.App;
+
+const sizeConfig = require('../private/size-config.json');
```

Comment:
private configs are just for dev (when running locally), it will not be available once uploaded to firebase